commands/acl-setuser.md

```diff
@@ -40,11 +53,14 @@ The following documentation is a reference manual about the capabilities of this
 
 ## List of rules
 
+Redis ACL rules are split into two categories, rules that define command permissions, "Command rules", and rules that define user state, "User management rules".
 This is a list of all the supported Redis ACL rules:
 
-* `on`: set the user as active, it will be possible to authenticate as this user using `AUTH <username> <password>`.
-* `off`: set user as not active, it will be impossible to log as this user. Please note that if a user gets disabled (set to off) after there are connections already authenticated with such a user, the connections will continue to work as expected. To also kill the old connections you can use `CLIENT KILL` with the user option. An alternative is to delete the user with `ACL DELUSER`, that will result in all the connections authenticated as the deleted user to be disconnected.
+### Command rules
+
 * `~<pattern>`: add the specified key pattern (glob style pattern, like in the `KEYS` command), to the list of key patterns accessible by the user. You can add multiple key patterns to the same user. Example: `~objects:*`
+* `%R~<pattern>`: Add the specified read key pattern. This behaves similar to the regular key pattern but only grants permission to read from keys that match the given pattern.
+* `%W~<pattern>`: Add the specified write key pattern. This behaves similar to the regular key pattern but only grants permission to write to keys that match the given pattern.
```

Comment:
1. we must specify that these rules are only supported from redis 7.0 and up.
2. i think we must give a link to the place that specifies what's a read, probably to the section i see you added in topics/acl (`https://redis.io/topics/key-specs#Key permissions`?)
3. i think we should change the above description of the `~<pattern>` to explicitly say it allows both read and write
4. i think it might be nice to also mention that `%RW~` is supported and is an alias for just `~`

these comments are probably applicable for topics/acl.md too